src/i18n/ar.json

```diff
@@ -2,5 +2,14 @@
   "home": {
     "title": "وطن",
     "welcome": "أهلًا وسهلًا"
+  },
+  "button": {
+    "title": "Click"
```

Comment:
Why are these in english and not arabic?

Reply:
Changed